Assets/Character/Scripts/Consumable.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Consumable : MonoBehaviour {
+
+    private int count;
+    private float cooldown;
+    private float startTime;
+    private bool healing;
+    private float duration;
+    private float interval;
+    private float timer;
```

Comment:
in general, a bunch of primitive member vars is pretty confusing and a sign that there is a better design